polygeist/lib/polygeist/Passes/ConvertPolygeistToLLVM.cpp

```diff
@@ -1374,27 +1374,14 @@ struct ConvertPolygeistToLLVMPass
       populatePolygeistToLLVMConversionPatterns(converter, patterns);
       populateSCFToControlFlowConversionPatterns(patterns);
       cf::populateControlFlowToLLVMConversionPatterns(converter, patterns);
-      populateMemRefToLLVMConversionPatterns(converter, patterns);
+      mlir::populateMemRefToLLVMConversionPatterns(converter, patterns);
```

Comment:
why do we need to qualify the namespace "mlir" here ? No other calls near this line do that.

Reply:
No need for that. I just named the new function the same first time and forgot to drop